assets/js/frontend/cart-fragments.js

```diff
@@ -38,6 +38,7 @@ jQuery( function( $ ) {
 	var $fragment_refresh = {
 		url: wc_cart_fragments_params.wc_ajax_url.toString().replace( '%%endpoint%%', 'get_refreshed_fragments' ),
 		type: 'POST',
+		timeout: 5000,
```

Comment:
Probably better make this dynamic, so people can filter this value, right now sounds too high.

Reply:
Is there an example in core for how to realize that? Is it a js var that will be populated by php so that a wordpress filter can be used?

Reply:
You can include here:

https://github.com/woocommerce/woocommerce/blob/master/includes/class-wc-frontend-scripts.php#L534-L541

You can use like:

```php
'request_timeout' => 5000,
```

And then use:

```js
timeout: wc_cart_fragments_params.request_timeout,
```